src/redis-cli.c

```diff
@@ -9418,9 +9606,10 @@ static long getLongInfoField(char *info, char *field) {
 }
 
 /* Convert number of bytes into a human readable string of the form:
- * 100B, 2G, 100M, 4K, and so forth. */
-void bytesToHuman(char *s, size_t size, long long n) {
+ * 1003B, 4.03K, 100.00M, 2.32G, 3.01T */
```

Comment:
why make effort to touch these? just to fill the floating part? doesn't feel very necessary.

Reply:
As I added Terabyte (T) in `bytesToHuman()', I wanted to show in the comment every possibilities. Moreover from the original comment we might think that we only convert to integer numbers, when in fact, even with the original version, we will display 100.00M.